.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   matrix:
     - GROWSTUFF_ELASTICSEARCH=true RSPEC_TAG=elasticsearch COVERAGE=true
     - GROWSTUFF_ELASTICSEARCH=false RSPEC_TAG=~elasticsearch COVERAGE=false
-    - STATIC_CHECKS=true
+    # - STATIC_CHECKS=true
```

Comment:
What's the reason for disabling these, do we need a ticket to remind us to turn them back on?

Reply:
I had to turn off the linters on travis, because overcommit isn't working (it depends on a gem that isn't maintained anymore, that is throwing errors at installation).